runtime/lua/vim/treesitter/_fold.lua

```diff
@@ -114,7 +114,7 @@ local function compute_folds_levels(bufnr, info, srow, erow, parse_injections)
   srow = srow or 0
   erow = erow or api.nvim_buf_line_count(bufnr)
 
-  local parser = assert(ts._get_parser(bufnr))
+  local parser = assert(ts.get_parser(bufnr, nil, { error = false }))
```

Comment:
Can this assert even be triggered, given that  `foldexpr` already returns early on the same check?

(That'd be fine and a good use of `assert` without error message; just wondering.)

Reply:
I think it maybe could, due to being called in the `autocmd` at the bottom. Not certain but maybe best to add an error message just in case